### DISCUSSION TITLE
Which commits/release is the hosted website https://agentgpt.reworkd.ai/

### DISCUSSION THREAD

**Discussion by yhyu13** [reworkd/AgentGPT#761] [Q&A] [answered]
Hi, I am trying to find a stable commit that matches the behavior/performance of the hosted website https://agentgpt.reworkd.ai/, which commit it is related to now? Is it always equal to the commit with the latest release tag?

**Comment by jasangill1** [comment, marked as answer]
Hello @yhyu13 that would be the most recent ones typically the release like beta v.0.8.0
